appeals/api/src/server/endpoints/site-visits/site-visits.validators.js

```diff
@@ -7,20 +7,15 @@ import validateTimeParameter from '#common/validators/time-parameter.js';
 import validateTimeRangeParameters from '#common/validators/time-range-parameters.js';
 import {
 	ERROR_INVALID_SITE_VISIT_TYPE,
-	ERROR_SITE_VISIT_REQUIRED_FIELDS,
-	SITE_VISIT_TYPE_UNACCOMPANIED
+	ERROR_SITE_VISIT_REQUIRED_FIELDS_ACCESS_REQUIRED,
+	SITE_VISIT_TYPE_UNACCOMPANIED,
+	SITE_VISIT_TYPE_ACCOMPANIED,
+	SITE_VISIT_TYPE_ACCESS_REQUIRED,
+	ERROR_SITE_VISIT_REQUIRED_FIELDS_ACCOMPANIED
 } from '#endpoints/constants.js';
-import checkStringsMatch from '#utils/check-strings-match.js';
 
 /** @typedef {import('express-validator').ValidationChain} ValidationChain */
 
-/**
- * @param {string} visitType
- * @returns {boolean}
- */
-const isVisitUnaccompanied = (visitType) =>
```

Comment:
I would have probably left this in, but  it does simplify the code, and this function is only used in one place, so I can see why you removed

Reply:
I did contemplate leaving this in but just renaming it (maybe adding a similar function for `isVisitAccompanied`). Btw, do you know if there's any advantage to using `checkStringsMatch` over a simple comparison using `===`?